index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Portfolio - Jeff Eng</title>
+    <link rel="stylesheet" href="vendor/styles/normalize.css">
+  </head>
+  <body>
+    <main>
+      <header>
+        <h1>Jeff Eng</h1>
+        <p>Aspiring JavaScript Web Developer</p>
+      </header>
+      <nav>
```

Comment:
You aren't closing this nav.  Also some of these other elements aren't being formatted correctly.  Go through lines 9-28 and reformat the structure of your elements.  It seems to all be jumbled up.